src/scripts/keyItems/KeyItems.ts

```diff
@@ -56,6 +56,9 @@ class KeyItems implements Feature {
             // TODO buy for 500 quest points
             new KeyItem(KeyItems.KeyItem.Event_calendar, 'This calendar will keep you up to date on the latest events', undefined, undefined, undefined, 'Event Calender'),
             new KeyItem(KeyItems.KeyItem.Shard_case, 'A case specifically designed for holding shards', undefined, undefined, undefined, 'Shard Case'),
+            new KeyItem(KeyItems.KeyItem.DNA_splicers, 'A splicer that fuses certain Pokémon.', () => {
```

Comment:
Removed the full stop to be consistent with the other key item descriptions.
```suggestion
            new KeyItem(KeyItems.KeyItem.DNA_splicers, 'A splicer that fuses certain Pokémon', () => {
```